site/layouts/partials/course_banner.html

```diff
@@ -1,5 +1,19 @@
+{{ $bannerClass := "text-uppercase display-4 font-weight-bold m-0 text-white" }}
+{{ $currentPage := . }}
 <div id="course-banner" class="p-0">
   <div class="max-content-width m-auto px-5 py-6">
-    <p class="text-uppercase display-4 font-weight-bold m-0">{{ .Params.course_title }}</p>
+    {{ if (isset $currentPage.Params "course_id") }}
```

Comment:
I think this should be true everywhere? But I'm fine to merge it and inline this part later if determine that that's the case

Reply:
yeah, I thought it would be too, but just added it because we have the same code in our nav partials